thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/job/JobScheduler.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.thirdeye.anomaly.job;
+
+import java.util.List;
+
+import org.quartz.SchedulerException;
+
+public interface JobScheduler {
```

Comment:
While not required, it might be more readable if the JobScheduler declares a generic type so it's clear what is being scheduled.